<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ArcGate2168" failures="0" tests="6" name="Testing.CheckKYC" time="0.022" errors="6" timestamp="2020-10-12T12:15:09 IST" skipped="0">
  <testcase classname="Testing.CheckKYC" name="check_kyc_verification_pending" time="0.007">
    <error message="Unresolved compilation problems: 
	privatekey04 cannot be resolved to a variable
	privatekey04 cannot be resolved to a variable
	The method getStatus() is undefined for the type BaseResponse
	The method getReference() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
	The method getStatus() is undefined for the type BaseResponse
	The method getReference() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
privatekey04 cannot be resolved to a variable
privatekey04 cannot be resolved to a variable
The method getStatus() is undefined for the type BaseResponse
The method getReference() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse
The method getStatus() is undefined for the type BaseResponse
The method getReference() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse

at Testing.CheckKYC.check_kyc_verification_pending(CheckKYC.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- check_kyc_verification_pending -->
  <system-out/>
  <testcase classname="Testing.CheckKYC" name="check_kyc_verification_successfully_passed" time="0.004">
    <error message="Unresolved compilation problems: 
	privatekey04 cannot be resolved to a variable
	The method getStatus() is undefined for the type BaseResponse
	The method getReference() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
	The method getStatus() is undefined for the type BaseResponse
	The method getReference() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
privatekey04 cannot be resolved to a variable
The method getStatus() is undefined for the type BaseResponse
The method getReference() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse
The method getStatus() is undefined for the type BaseResponse
The method getReference() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse

at Testing.CheckKYC.check_kyc_verification_successfully_passed(CheckKYC.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- check_kyc_verification_successfully_passed -->
  <system-out/>
  <testcase classname="Testing.CheckKYC" name="check_kyc_with_unregistered_user_handle" time="0.003">
    <error message="Unresolved compilation problems: 
	privatekey02 cannot be resolved to a variable
	The method getStatus() is undefined for the type BaseResponse
	The method getReference() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
	The method getStatus() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
privatekey02 cannot be resolved to a variable
The method getStatus() is undefined for the type BaseResponse
The method getReference() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse
The method getStatus() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse

at Testing.CheckKYC.check_kyc_with_unregistered_user_handle(CheckKYC.java:70)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- check_kyc_with_unregistered_user_handle -->
  <system-out/>
  <testcase classname="Testing.CheckKYC" name="check_kyc_with_empty_user_handle" time="0.003">
    <error message="Unresolved compilation problems: 
	privatekey02 cannot be resolved to a variable
	The method getStatus() is undefined for the type BaseResponse
	The method getReference() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
	The method getStatus() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
privatekey02 cannot be resolved to a variable
The method getStatus() is undefined for the type BaseResponse
The method getReference() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse
The method getStatus() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse

at Testing.CheckKYC.check_kyc_with_empty_user_handle(CheckKYC.java:93)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- check_kyc_with_empty_user_handle -->
  <system-out/>
  <testcase classname="Testing.CheckKYC" name="check_kyc_with_invalid_invalid_signature" time="0.003">
    <error message="Unresolved compilation problems: 
	privatekey02 cannot be resolved to a variable
	The method getStatus() is undefined for the type BaseResponse
	The method getReference() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
	The method getStatus() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
privatekey02 cannot be resolved to a variable
The method getStatus() is undefined for the type BaseResponse
The method getReference() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse
The method getStatus() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse

at Testing.CheckKYC.check_kyc_with_invalid_invalid_signature(CheckKYC.java:115)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- check_kyc_with_invalid_invalid_signature -->
  <system-out/>
  <testcase classname="Testing.CheckKYC" name="check_kyc_with_not_requested_kyc_user_handle" time="0.002">
    <error message="Unresolved compilation problems: 
	privatekey05 cannot be resolved to a variable
	The method getStatus() is undefined for the type BaseResponse
	The method getReference() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
	The method getStatus() is undefined for the type BaseResponse
	The method getMessage() is undefined for the type BaseResponse
	ErrorCheckKYCMessage2 cannot be resolved to a variable
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
privatekey05 cannot be resolved to a variable
The method getStatus() is undefined for the type BaseResponse
The method getReference() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse
The method getStatus() is undefined for the type BaseResponse
The method getMessage() is undefined for the type BaseResponse
ErrorCheckKYCMessage2 cannot be resolved to a variable

at Testing.CheckKYC.check_kyc_with_not_requested_kyc_user_handle(CheckKYC.java:138)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- check_kyc_with_not_requested_kyc_user_handle -->
  <system-out/>
</testsuite> <!-- Testing.CheckKYC -->
